alf/algorithms/entropy_target_algorithm.py

```diff
@@ -144,6 +168,19 @@ def calc_loss(self, training_info: EntropyTargetInfo):
         avg_entropy = self._avg_entropy.average(entropy)
 
         def _init():
+            below = avg_entropy < self._max_entropy
+            increasing = tf.cast(avg_entropy > prev_avg_entropy, tf.float32)
+            # -1 * increasing + 0.5 * (1 - increasing)
+            update_rate = (
+                0.5 - 1.5 * increasing) * self._very_slow_update_rate
+            # cannot use tf.cond because the two return types are different
+            run_if(below, lambda: self._stage.assign(-1))
+            run_if(
```

Comment:
We can simply the code by removing run_if. An extra ajust of log_alpha does not matter.